Quadriga/src/main/java/edu/asu/spring/quadriga/domain/impl/ConceptpowerReply.java

```diff
@@ -96,7 +99,29 @@
     public void setConceptEntry(List<ConceptpowerReply.ConceptEntry> entries) {
         this.conceptEntry = entries;
     }
+    
+    @XmlElement(required = true, namespace="http://www.digitalhps.org/")
+    protected List<ConceptpowerReply.AlternativeIds> alternativeIds;
```

Comment:
There is one alternativeIds entry for each conceptEntry in ConceptpowerReply. As conceptEntry is defined as List, alternativeIds is defined as list.